### PR TITLE
fix : fix layout shift in navigating

### LINKED ISSUE #36: layout shift(width) on navigating other pages
layoutshift

### PR DESCRIPTION
**Pull request by anonymousRecords**
### Description
- close #36 
- remove `overflow-y: scroll;` in body

### CODE REVIEW

File: components/ui/header.tsx [hunk 1]
```diff
@@ -79,7 +68,7 @@ export default function Header() {
       {/* Drawer */}
       <div
         className={cn(
-          "fixed top-0 right-0 bottom-0 w-full bg-white dark:bg-black z-50 p-6 transform transition-transform duration-300 ease-in-out md:hidden",
+          "fixed top-0 right-0 bottom-0 w-full bg-white dark:bg-black z-50 p-6 transform transition-transform duration-300 ease-in-out md:hidden ov",
```
Comment (Moon-DaeSeung) [inline components/ui/header.tsx:71]: typo error

File: app/globals.css [hunk 2]
```diff
@@ -104,8 +101,7 @@ h6 {
 
 /* For WebKit browsers (Chrome, Safari) */
 ::-webkit-scrollbar {
-  width: 8px;
-  height: 8px;
+  width: 6px;
```
Comment (Moon-DaeSeung) [inline app/globals.css:104]: p2: why change style?

if you want better style, you should give anotion pr

File: app/globals.css [hunk 3]
```diff
@@ -48,8 +47,6 @@ ol {
 /* Global Styles */
 body {
   line-height: 1.6;
-  background-color: #f8f9fa;
```
Comment (Moon-DaeSeung) [inline app/globals.css:51]: p2: why change style?

if you want better style, you should give anotion pr

File: components/ui/header.tsx [hunk 4]
```diff
@@ -14,17 +14,6 @@ import ThemeSelector from "./theme-selector";
 export default function Header() {
   const [isMenuOpen, setIsMenuOpen] = useState<boolean>(false);
 
-  useEffect(() => {
-    if (isMenuOpen) {
-      document.body.style.overflow = "hidden";
```
Comment (Moon-DaeSeung) [inline components/ui/header.tsx:19]: p2: you just delete it.

how can you prevent scrolling when mobile menu open

File: app/globals.css (hunk 3 above)
Comment (anonymousRecords) [reply app/globals.css:51]: In the case of Windows, I noticed that the transparent property for `::-webkit-scrollbar-trac` is not applied(transparent) due to the background color.

I will work on it with a screenshot in the next PR.

**Pull request closed without merging**